src/jvm.rs

```diff
@@ -135,6 +135,56 @@ fn get_or_default_init_jvm() -> crate::Result<JvmPtr> {
     }
 }
 
+fn throw_java_runtime_exception(env: EnvPtr<'_>, message: &str) {
+    let mut jvm = Jvm(env);
+
+    let runtime_exception_clazz = crate::java::lang::RuntimeException::class(&mut jvm)
+        .expect("java/lang/RuntimeException not found");
+
+    let runtime_exception_clazz_ptr = runtime_exception_clazz.as_raw().as_ptr();
+
+    let encoded = cesu8::to_java_cesu8(message);
+    // SAFETY: cesu8 encodes interior nul bytes as 0xC080
+    let c_string = unsafe { CString::from_vec_unchecked(encoded.into_owned()) };
+    let c_string_ptr = c_string.as_ptr();
+
+    unsafe {
+        env.invoke_unchecked(
+            |env| env.ThrowNew,
+            |jni, f| f(jni, runtime_exception_clazz_ptr, c_string_ptr),
+        );
+    };
+}
+
+fn error_to_java_exception(env: EnvPtr<'_>, error: Error<Local<Throwable>>) {
```

Comment:
```suggestion
fn error_to_java_exception(env: EnvPtr<'_>, error: Error<Local<'_ Throwable>>) {
```

I prefer to make these lifetimes explicit, allowing me to distinguish borrowed from fully owned types at a glance.